Improve documentation after defining API (#12)
kyma-incubator/octopus: docs/08-tutorial.md

## Tutorial

> In this tutorial, you create `TestDefinition` and `ClusterTestSuite` to show Octopus in action.

Reviewer: ```suggestion
Follow this tutorial to create `TestDefinition` and `ClusterTestSuite` CRDs and to see Octopus in action.
```

> 1. Prepare data

Reviewer: ```suggestion
1. Create `TestDefinition` and `ClusterTestSuite` CRDs:
```

> Create `TestDefinition` and `ClusterTestSuite`:

Reviewer: ```suggestion

```

```bash
cd $GOPATH/src/github.com/kyma-incubator/octopus
kubectl apply -f config/samples/testdefinition.yaml
kubectl apply -f config/samples/testsuite.yaml

```

> Created `TestDefinition` is very simple, it just executes `pwd` command in the `alpine:latest` 

Reviewer: Not sure whether this sentence is necessary.

Author: removed

image, so it should always succeed.  
 

2. Run the controller:
```
cd $GOPATH/src/github.com/kyma-incubator/octopus
make run
```

> Then you should see in the logs similar output:

Reviewer: ```suggestion
The sample output looks as follows:
```

Author: we should have a dictionary with common phrases

```
{"level":"info","ts":...,"logger":"cts_controller","msg":"Initialize suite","suite":"testsuite-all"}
{"level":"info","ts":...,"logger":"cts_controller","msg":"Testing pod created","suite":"testsuite-all","podName":"octopus-testing-pod-jz9qq","podNs":"default"}
{"level":"info","ts":...,"logger":"cts_controller","msg":"Do nothing, suite is finished","suite":"testsuite-all"}
```

> After controller finished processing created test suite, the suite is marked as a `Succeeded`:

Reviewer: ```suggestion
After the controller finishes processing the created test suite, the suite is marked as a **Succeeded**:
```

`kubectl get ClusterTestSuite testsuite-all -oyaml`

```
apiVersion: testing.kyma-project.io/v1alpha1
kind: ClusterTestSuite
metadata:
  labels:
    controller-tools.k8s.io: "1.0"
  name: testsuite-all
  uid: ba1a98e3-3ef5-11e9-b7e0-3af374a09d13
spec:
  maxRetries: 1
status:
  completionTime: 2019-03-06T12:37:20Z
  conditions:
  - status: "False"
    type: Running
  - status: "True"
    type: Succeeded
  results:
  - executions:
    - completionTime: 2019-03-06T12:37:20Z
      id: octopus-testing-pod-jz9qq
      podPhase: Succeeded
      startTime: 2019-03-06T12:37:15Z
    name: test-kubeless
    namespace: default
    status: Succeeded
  startTime: 2019-03-06T12:37:15Z

```

> To learn which testing pods were created, execute the following command:

Reviewer: ```suggestion
To learn which testing Pods were created, run:
```
